packages/storybook/src/nimble/text-field/text-field-matrix.stories.ts

```diff
@@ -111,6 +114,7 @@ const component = (
         ?readonly="${() => readonly}"
         ?error-visible="${() => errorVisible}"
         error-text="${() => errorText}"
+        ?appearance-readonly="${() => appearanceReadonly}"
```

Comment:
It's a pain but lets only ever use `readonly` for the attribute and consistently use readOnly for JS properties

```suggestion
        ?appearance-readonly="${() => appearanceReadOnly}"
```